src/middlewared/middlewared/plugins/kubernetes_linux/update.py

```diff
@@ -132,8 +157,12 @@ async def do_update(self, job, data):
         old_config.pop('dataset')
         config = old_config.copy()
         config.update(data)
+        migrate = config.get('migrate_applications')
+
+        await self.validate_data(config, 'kubernetes_update', old_config)
```

Comment:
I don't see you checking if there are verrors neither there nor here

Reply:
We raise verrors at the end of `validate_data`